server/routes/tasks.routes.ts

```diff
@@ -82,13 +107,60 @@ router.post(
       await connection.end();
       res
         .status(200)
-        .json({ success: true, event_id: resultsInsertTask.insertId });
+        .json({ success: true, task_id: resultsInsertTask.insertId });
```

Comment:
I altered the naming here since we're adding a task not event, not sure if this will cause problems and frontend might need to be adjusted for this key